src/loaders/db.js

```diff
@@ -0,0 +1,20 @@
+const pg = require('pg');
+const fs = require('fs-extra');
+
+const conString = "postgres://lzxdfpik:bILzLD1yrycRbYZb74fckcKt2tJTiqZF@balarama.db.elephantsql.com:5432/lzxdfpik";
```

Comment:
need to move connection string to config, it's can be done after approve this PR https://github.com/TalkyEk/Yelanam-back/pull/13